src/metabase/email/messages.clj

```diff
@@ -291,6 +291,36 @@
                :message      (stencil/render-file "metabase/email/follow_up_email" context)}]
     (email/send-message! email)))
 
+(defn send-creator-sentiment-email!
+  "Format and send an email to the system admin following up on the installation."
+  [{:keys [email created_at first_name num_dashboards num_questions num_models]} instance-data]
+  {:pre [(u/email? email)]}
+  (let [blob (-> {:instance instance-data
```

Comment:
nit: `blob` only needs to be computed when anon tracking is enabled... maybe make it a function